scripting/get5/matchconfig.sp

```diff
@@ -277,6 +277,16 @@ stock bool LoadMatchFromUrl(const char[] url, ArrayList paramNames = null,
       return false;
     }
 
+    LogDebug("token (SteamWorks) = %s", token);
+
+    if(strlen(token) > 0){
+      char cleanedToken[1024];
+      strcopy(cleanedToken, sizeof(cleanedToken), token);
+      Format(cleanedToken, sizeof(cleanedToken), "Bearer %s", cleanedToken);
```

Comment:
This `cleanedToken` part is also meaningless. Nothing is being cleaned; it's just copied. You could achieve the same with just:

```c
char formattedToken[1024];
Format(formattedToken, sizeof(formattedToken), "Bearer %s", token);
```